include/swift/Serialization/SerializedSILLoader.h

```diff
@@ -112,6 +112,10 @@ class SerializedSILLoader {
 
   /// Deserialize all SILFunctions, VTables, and WitnessTables for
   /// a given Module.
+  void getAllForModule(Identifier Mod);
```

Comment:
This is the same as passing `nullptr` for `PrimaryFile` below. I'm not sure if this is better because it's more explicit, or worse because it's easy to do by accident.